DataFormats/SoATemplate/interface/SoALayout.h

```diff
@@ -424,7 +424,7 @@
  */
 // clang-format off
 #define GENERATE_SOA_LAYOUT(CLASS, ...)                                                                                \
-  template <CMS_SOA_BYTE_SIZE_TYPE ALIGNMENT = cms::soa::CacheLineSize::defaultSize,                                   \
+  template <cms::soa::size_type ALIGNMENT = cms::soa::CacheLineSize::defaultSize,                                      \
```

Comment:
Not sure if this has any relation to the dictionary problem, but should this be
```suggestion
  template <cms::soa::byte_size_type ALIGNMENT = cms::soa::CacheLineSize::defaultSize,                                      \
```
? (currently this PR changes the underlying type from `std::size_t` to `cms_int32_t`)